application/ai/tension_scoring_contract.py

```
@@ -36,6 +36,16 @@ class TensionScoringLlmPayload(BaseModel):
     emotional_justification: str = Field(default="", max_length=500)
     pacing_justification: str = Field(default="", max_length=500)
 
+    @field_validator("plot_tension", "emotional_tension", "pacing_tension", mode="before")
+    @classmethod
+    def coerce_to_float(cls, v):
+        if isinstance(v, str):
+            try:
+                return float(v)
+            except ValueError:
+                raise ValueError(f"无法将 '{v}' 转为数字")
```

Ruff (0.15.10)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)